src/bootstrap/src/core/download.rs

```diff
@@ -229,7 +232,7 @@ impl Config {
             tempfile.to_str().unwrap(),
             "--retry",
             "3",
-            "-SRf",
+            "-SRfL",
```

Comment:
The new option `-L` is because the current download link needs to follow a redirection for some reasons...